tuf/api/metadata.py

```diff
@@ -965,12 +922,11 @@ class Delegations:
     """A container object storing information about all delegations.
 
     Attributes:
-        keys: A dictionary of keyids and key objects containing information
-            about the corresponding key.
-        roles: A list of DelegatedRole instances containing information about
-            all delegated roles.
+        keys: Dictionary of keyids to Keys. Defines the keys used in 'roles'.
+        roles: List of DelegatedRoles that define which keys are required to
+            sign the metadata for a specific role. The roles order also
+            defines the order the role delegations are considered.
```

Comment:
```suggestion
            defines the order that role delegations are considered in.
```
?

I wanted to say "in which the role delegations are considered" but thought you might object to the character count increase.